3rdparty/CMakeLists.txt

```diff
@@ -3,6 +3,19 @@
 
 # Note: order does matter
 
+if (WIN32)
+  find_program(BASH bash)
```

Comment:
@andschwa - I wasn't done with my changes when you reviewed it. As you noticed on my system which had both WSL bash and Git bash installed, find_program reported the Git bash as the program found instead of WSL bash i.e. it reported the last path found. Instead of putting convoluted logic to find all the paths, the previous logic of finding Git Bash explicitly on Windows should work for now. Reverting to that...